mediapicker/src/main/java/com/mediapicker/gallery/presentation/fragments/HomeFragment.kt

```diff
@@ -194,19 +202,21 @@ open class HomeFragment : BaseFragment() {
     }
 
     private fun setUpWithTabLayout() {
-        PagerAdapter(
-            childFragmentManager, listOf(
-                PhotoGridFragment.getInstance(
-                    getString(R.string.oss_title_tab_photo),
-                    getPhotosFromArguments()
-                ),
-                VideoGridFragment.getInstance(
-                    getString(R.string.oss_title_tab_video),
-                    getVideosFromArguments()
+        childView.findViewById<ViewPager>(R.id.viewPager).apply {
```

Comment:
I think we can remove this line `childView.findViewById<ViewPager>(R.id.viewPager).apply {`

Reply:
need to save `viewPager` as variable first, done as well.